addons/im_livechat/models/mail_channel.py

```diff
@@ -88,10 +88,10 @@ def _channel_get_livechat_visitor_info(self):
         self.ensure_one()
         # remove active test to ensure public partner is taken into account
         channel_partner_ids = self.with_context(active_test=False).channel_partner_ids
-        partners = channel_partner_ids - self.livechat_operator_id
+        partners = channel_partner_ids - self.livechat_operator_id - self.env.ref('base.partner_root')
```

Comment:
This shouldn't be needed, it means demo are adding the `partner_root` when it should not.